flexmeasures/data/models/charts/belief_charts.py

```diff
@@ -75,38 +83,68 @@ def bar_chart(
 
 
 def chart_for_multiple_sensors(
-    sensors: list["Sensor"],  # noqa F821
+    sensors_to_show: list["Sensor", list["Sensor"]],  # noqa F821
     event_starts_after: datetime | None = None,
     event_ends_before: datetime | None = None,
     **override_chart_specs: dict,
 ):
-    sensors_specs = []
+    # Unpack nested sensors
+    sensors = flatten_unique(sensors_to_show)
```

Comment:
Small point: I believe you're only using `sensors` in the very next step (to determine the shared data resolution), so you could also flatten them in line 97. Or move this line under the comment in line 94 that says what is happening.